solutions/kubernetes/functions.sh

```diff
@@ -18,6 +18,11 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+function add_separator {
+    set +x +e
```

Comment:
can't we combine both flags as `set +xe`?

Reply:
Done